examples/SingleParticleCurrent/include/simulation_defines/unitless/physicalConstants.unitless

```diff
@@ -97,4 +97,13 @@ namespace picongpu
     // = 1/c^2
     const float_X MUE0_EPS0 = float_X(1. / SPEED_OF_LIGHT / SPEED_OF_LIGHT);
 
+    /* Atomic unit of electric field in PIC Efield units */
+    const float_X ATOMIC_UNIT_EFIELD = float_X(SI::ATOMIC_UNIT_EFIELD / UNIT_EFIELD);
+
+    /* Atomic unit of time in PIC units */
+    const float_X ATOMIC_UNIT_TIME = float_X(SI::ATOMIC_UNIT_TIME / UNIT_TIME);
+
+    /* Time step of the simulation in atomic units */
+    const float_X DELTA_T_AU = float_X(SI::DELTA_T_SI / SI::ATOMIC_UNIT_TIME);
```

Comment:
Probably the compiler would optimize that away, yet, is that really necessary?
The time step of the simulation is known at compile time and being in atomic units makes it a unitless number. 
Sure it is not in PIC units but I don't need it in PIC units there. 

I didn't even like writing `ATOMIC_UNIT_TIME` in PIC units because it seems a bit misleading to me to use something like that. I just might need it later when I do averaging over time steps amd make use of `currentStep`.